src/popup/locales/en.json

```
@@ -581,6 +581,7 @@
         "mobile-app": "Mobile app"
       },
       "webVersion": "Or continue with our web version",
+      "mobileVersion": "More than just a way to receive, store & send tokens on the aeternity blockchain",
```

```suggestion
      "mobileVersion": "More than just a way to receive, store & send coins on the aeternity blockchain",
```
They rather meant AE then actual tokens 🤔